RELEASES.md

```diff
@@ -6,6 +6,12 @@ The list below shows the additions, changes and fixes that are visible to users
 
 ## Compatibility
 
+Make sure to use a recent Rust nightly and `cargo-contract` with the current
+release: 
+```
+cargo install --force cargo-contract && rustup update
```

Comment:
I'm not 100% sure about adding `--force`, it would mean if someone had installed from Github with the same version it would be overwritten. Which may or may not be what they want.

Reply:
I've set it to the current fixed version.